source/particle/property/cpo_elastic_tensor.cc

```diff
@@ -0,0 +1,313 @@
+/*
+  Copyright (C) 2022 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/cpo_elastic_tensor.h>
+#include <aspect/particle/world.h>
+
+#include <aspect/utilities.h>
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Property
+    {
+
+
+      template <int dim>
+      CpoElasticTensor<dim>::CpoElasticTensor ()
+      {
+        permutation_operator_3d[0][1][2]  = 1;
```

Comment:
I think it would be useful to make a standard Levi-Civita tensor somewhere.

Reply:
See #5318